### ISSUE TITLE
Encouragements for current and former NVDA contributors

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#10000]
Dear NVDA contributors,

At last - we can say we hit 10000 GitHub issues. Although it is an interesting milestone, it still shows how much work we need to do.

To former contributors, thank you for your hard work over the year in laying foundations for NVDA today. Your contributions are not forgotten.

To current contributors, please keep up the good work you've been doing for a while.

Thanks.

**Comment by Adriani90** [comment]
And a very good proof of the enormous effort put in this Project can be seen in the pull requests, which also exceeded 10 thousand. Good job guys!

**Comment by DrSooom** [comment]
@Adriani90: Sorry, but that's not the case. Issues and Pull Requests on GitHub use the same numbering. Therefore this issue here isn't the 10,000th issue, it only has the number #10000.

At the moment (2019-07-31 19:22 UTC+2) we have:
+ Issues: 2,233 open; 6,732 closed; 8,965 total
+ PRs: 72 open; 961 closed; 1,033 total
+ Total: 9,998; Deleted: 10; Highest number: #10008

**Comment by derekriemer** [comment]
This is a milestone to be proud of. As a product that has improved the lives of over 100,000 people, we should be proud of the work that has been accomplished. The NVDA project has helped transform the industry from one controlled by near monopolies, and brought competition to those monopolies. Thanks to NV Access's hard work over the last 13 years, Especially the work of @michaelDCurran @jcsteh, @feerrenrut, and @leonardder blind people are not forced to pay thousands of dollars to gain independent access to technological solutions that are necessary for employment and equal opportunity. While the bulk of the work behind this project has been accomplished by NV Access and @leonardder, nothing we have would be possible without the effort put in by many contributors. @josephsl deserves special thanks. Some of his accomplishments include: creator of NVDACon, a Korean translator for over 7 years,   author of incredibly many add-ons, an addon reviewer and website maintainer, owner of many lists to encourage blind developers, someone offering technical help to NVDA developers, a major contributor and driver behind much of NVDA's python3 transition, experimenting with wxPython updates, principal designer of automatic NVDA addon updates which is an ongoing project, multi-year moderator on the NVDA mailing list, mentor to several NVDA developers, champion for windows 10 a11y progress, and more.
As a testament to how many people the NVDA project has inspired, one must simply open the contributors file. The list of people and organizations that have directly contributed code or translations to NVDA is 233 as of August of 2019, and that doesn't mention all of the people who have contributed through mailing list submissions, NVDACon talks, and other advocacy work. Keeping the project alive requires many people, including developers, translators, alpha and beta testers, developers, documentation authors, technical support volunteers, and evangelists. I want to take this moment to thank each and every one of you for your efforts both big and small to this project. When you are struggling with an accessibility issue, remember how far we have come by simply embracing the ideal of blind people building their own technology. Keep your thoughts positive, because each small step made on assistive technology requires having the courage to solve problems that often are frustrating.
As of August of 2019, NVDA has an average daily count of  33800 users. The real number of users is much higher, because many people do not have reliable internet access, or have disabled user statistics gathering.

**Comment by Cleversn** [comment]
Yes, thanks all. being one of the first translators since 2006, of course I'm glad as well for this achievement.

**Comment by zahra21** [comment]
hi.
i always remember your great work and your kindness.
i always pray for you and request divine infinite mercy, blessings, favours, bounties and graces.
nvda has numerous blessings for me till the end of my life and even afterlife!
that i am sure its one of the numerous blessings and divine mercy for me,
and contribution and great efforts of its developers.
God bless you all nvda developers, programmers, users and supporters!

**Comment by Adriani90** [comment]
I am closing this issue for now since we are almost at 11.000.
